core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceRequestConverterTest.java

```diff
@@ -513,6 +521,40 @@ public void validate() {
         }
     }
 
+    static class RequestBean4 {
+        private int foo;
+
+        // This field is not set because the foo param is already used in the constructor.
+        @Param("foo")
+        private int foo1;
```

Comment:
- Could you update `RequestBean2`?
- How about adding a test case that makes sure `@Header("foo")` and `@Param("foo")` do not cause a problem?

Reply:
Updated Thanks~!